src/components/views/read/cvForm/PersonalInfo.tsx

```diff
@@ -0,0 +1,350 @@
+        )?.personalLink || '',
+      'Personal Picture':
+        inputValues.find(
+          (inputValue) => Object.keys(inputValue)[0] === 'preview',
+        )?.preview || '',
+    };
+
+    handleValues('Personal Info', [modifiedValues]);
+    nextPage();
+    nextStep();
+  };
+  return (
+    <div>
+      <div>
+        <form
+          style={{
+            position: 'absolute',
+            top: '305px',
+            left: '528px',
+            minWidth: '120px',
+          }}
+        >
+          <p>First Name</p>
+          <TextField
+            label="First Name"
+            id="firstName"
+            value={
+              inputValues.find(
+                (inputValue) => Object.keys(inputValue)[0] === 'firstName',
+              )?.firstName || ''
+            }
+            onChange={(e) => handleChange('firstName', e.target.value)}
+            required
+          />
+          <p>Last Name</p>
+          <TextField
+            label="Last Name"
+            id="lastName"
+            value={
+              inputValues.find(
+                (inputValue) => Object.keys(inputValue)[0] === 'lastName',
+              )?.lastName || ''
+            }
+            onChange={(e) => handleChange('lastName', e.target.value)}
+            required
+          />
+          <p>Date of Birth</p>
+          <LocalizationProvider dateAdapter={AdapterDayjs}>
+            <DatePicker
+              label="Birth Date"
+              value={birthDate}
+              maxDate={dayjs()}
+              onChange={(date) => {
+                const formattedDate = date
+                  ? dayjs(date).format('YYYY-MM-DD')
+                  : '';
+                setBirthDate(date);
+                setInputValues((prevInputValues) => [
+                  ...prevInputValues.filter(
+                    (inputValue) => Object.keys(inputValue)[0] !== 'birthDate',
+                  ),
+                  { birthDate: formattedDate },
+                ]);
+              }}
+            />
+          </LocalizationProvider>
+          <p>Gender</p>
+          <TextField
+            id="select-gender"
+            select
+            label="Gender"
+            value={
+              inputValues.find(
+                (inputValue) => Object.keys(inputValue)[0] === 'gender',
+              )?.gender || ''
+            }
+            onChange={(e) => handleChange('gender', e.target.value)}
+            required
+            helperText="Please select your gender"
+            margin="normal"
+          >
+            <MenuItem value="female">Female</MenuItem>
+            <MenuItem value="male">Male</MenuItem>
+            <MenuItem value="noIndicate">Do not Indicate</MenuItem>
+          </TextField>
+          <div
+            style={{
+              position: 'absolute',
+              top: '10px',
+              left: '328px',
+              // display: 'flex',
+            }}
```

Comment:
Remove.

I see you are splitting your form into 3 chunks that you then absolutely place on the page. You are looking for something like flex-wrap or a simple multi- column layout using nested containers in opposite direction.